string-manipulation-3/src/is-anagram.js

```diff
@@ -0,0 +1,14 @@
+/* exported isAnagram */
+
+function isAnagram(firstString, secondString) {
+  var firstNoSpace = firstString.replaceAll(' ', '');
+  var secondNoSpace = secondString.replaceAll(' ', '');
+  var counter = 0;
+
+  for (var i = 0; i < secondNoSpace.length; i++) {
+    if (firstNoSpace.includes(secondNoSpace[i])) {
+      counter++;
+    }
+  }
+  return counter === firstNoSpace.length;
```

Comment:
This is close, but will return `true` for `"red dad"` and `"rad ede"`. The number of times a letter occurs is significant. Its inclusion isn't enough.